packages/harmony/src/components/button/Button/Button.stories.tsx

```diff
@@ -80,23 +54,169 @@ export const Sizes: Story = {
   )
 }
 
-// Primary w/ color
-export const PrimaryWithColor: Story = { args: { hexColor: '#13C65A' } }
+export const States: Story = {
+  render: () => (
+    <Flex
+      direction='column'
+      gap='2xl'
+      justifyContent='center'
+      alignItems='flex-start'
+    >
+      <Flex gap='2xl' alignItems='center'>
+        <Text css={{ width: 56 }}>Hover</Text>
```

Comment:
why do we need width on these texts?

Reply:
harmony docs just to make sure the text is set at same point so the buttons align correctly